zerver/openapi/python_examples.py

```diff
@@ -309,48 +309,54 @@ def get_invitations(client: Client) -> None:
     # Get all invitations
     result = client.call_endpoint(url="/invites", method="GET")
     # {code_example|end}
-
+    assert result["result"] == "success"
     validate_against_openapi_schema(result, "/invites", "get", "200")
 
 
 @openapi_test_function("/invites:post")
 def send_invitations(client: Client) -> None:
     # {code_example|start}
     # Send invitations
+    streams = client.get_subscriptions()
+    stream_ids = [stream["stream_id"] for stream in streams["subscriptions"]][:3]
     request = {
         "invitee_emails": "example@zulip.com, logan@zulip.com",
         "invite_expires_in_minutes": 60 * 24 * 10,  # 10 days
         "invite_as": 400,
-        "stream_ids": [1, 11, 12],
+        "stream_ids": stream_ids,  # [10, 12, 3]
```

Comment:
So I just want to emphasize that the block inside the `{code_example|start}` tags is what appears in the user-facing documentation. So generally we're going to want to put any preparatory requests outside those blocks, and try to make those examples look as clean/clear as possible.

Reply:
I don't think we actually care what the IDs are, so I don't think the comment is required?

Reply:
I actually added that comment to emphasize that `stream_ids` is a list. Should I remove it?

Reply:
OK, I have removed that comment.